admin-server/src/main/kotlin/org/stellar/admin/server/config/RootConfig.kt

```diff
@@ -0,0 +1,9 @@
+package org.stellar.admin.server.config
+
+class RootConfig {
```

Comment:
What is this config used for?

Reply:
This is temporary. I plan to move it to a shared sub-project when refactoring the other backend servers.